bin/spice/cmd/catalogs.go

```diff
@@ -51,5 +56,6 @@ spice catalogs
 
 func init() {
 	catalogsCmd.Flags().String("tls-root-certificate-file", "", "The path to the root certificate file used to verify the Spice.ai runtime server certificate")
+	catalogsCmd.Flags().String("api-key", "", "The API key to use for authentication")
```

Comment:
You can use `RootCmd.PersistentFlags().String("api-key", "", "The API key to use for authentication")` to avoid defining the flag for each CLI command?

Reply:
Not all of the CLI commands make sense to have an API key though